src/test/java/com/placelab/tests/NegativeTestEmptyPasswordField.java

```diff
@@ -0,0 +1,63 @@
+package com.placelab.tests;
+
+import com.placelab.utils.WebDriverSetup;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
+public class NegativeTestEmptyPasswordField {
+    private WebDriver driver;
+
+    @Parameters("browser")
+    @BeforeTest
+    public void setUpDriver(String browser) {
+        this.driver = WebDriverSetup.getWebDriver(browser);
+        driver.get("https://demo.placelab.com");
+    }
+
+    @Parameters
+    @BeforeTest
+    private void validateLogInPage() {
+//        In real life there would be separate test to validate full content and display of log in page
+//        Should I do this separately with validation for every element of the page?
```

Comment:
it should be done, but if you want to we can do this as part of the next homework, it is good that you noticed